model/note/text.go

```diff
@@ -0,0 +1,104 @@
+package note
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/cozy/prosemirror-go/markdown"
+	"github.com/cozy/prosemirror-go/model"
+)
+
+func textSerializer() *markdown.Serializer {
+	nodes := map[string]markdown.NodeSerializerFunc{
+		"paragraph": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderInline(node)
+			state.CloseBlock(node)
+		},
+		"text": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.Text(*node.Text)
+		},
+		"bulletList": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderList(node, "", func(_ int) string { return "- " })
+		},
+		"orderedList": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderList(node, "", func(i int) string { return fmt.Sprintf("%d.", i+1) })
+		},
+		"listItem": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderContent(node)
```

Comment:
Maybe we should use `state.RenderInline()` to avoid empty lines in lists?

Reply:
I think it doesn't work when we have nested nodes inside the `listItem`

Reply:
Maybe, but I'm not really inspired by this...